### PR TITLE
script for ci

### PR DESCRIPTION
**Pull request by UncP**

### CODE REVIEW

File: tidb/run.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+rm failed.log
+
+for test in "bank" "sets" "register"
+do
+    for nemesis in "none" "parts" "majority-ring" "start-stop-2" "start-kill-2"
+        do
+	        lein run test --test ${test} --nemesis ${nemesis} --time-limit 60 --recovery-time 30 --concurrency 10
```
Comment (siddontang) [inline tidb/run.sh]: run 300s

File: tidb/run.sh [hunk 2]
```diff
@@ -2,18 +2,29 @@
...
             if [ $? -ne 0 ]
             then
                 echo ${test} ${nemesis} >> failed.log
             fi
     done
 done
 
+for nemesis in "none" "parts" "majority-ring" "start-stop-2"
+do
+    if [ ${nemesis} = "start-stop-2" ]
```
Comment (siddontang) [inline tidb/run.sh]: why 30 or 60 here? IMO, you can use  180 for all.

Comment (UncP) [inline tidb/run.sh]: this loop is for register test, it might run out of memory if we choose 180, i will try it anyway.

File: tidb/run.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+rm failed.log
+
+for test in "bank" "sets"
+do
+    for nemesis in "none" "parts" "majority-ring" "start-stop-2" "start-kill-2"
+        do
+	        lein run test --test ${test} --nemesis ${nemesis} --time-limit $t --concurrency 10
+            if [ $? -ne 0 ]
+            then
+                echo ${test} ${nemesis} >> failed.log
```
Comment (siddontang) [inline tidb/run.sh]: echo the test and nemesis, then exit directly, no need to output to failed.log.

**Comment by siddontang** [comment]
you can use a function like `get_time_limit(case, nemeis)` , and for register and start-stop-2, you can use 60, and use 300 for others.

File: tidb/run.sh [hunk 4]
```diff
@@ -1,39 +1,33 @@
...
-rm failed.log
+get_time() {
+    if [ $1 = "register" ]
+    then
+        if [ $2 = "start-stop-2" ]
+        then
+            return 60
+        elif [ $2 = "start-kill-2" ]
+        then
+            return 0
```
Comment (siddontang) [inline tidb/run.sh]: why 0 here?

File: tidb/run.sh [hunk 5]
```diff
@@ -0,0 +1,42 @@
+#! /bin/sh
+
+# for bank and sets test, we run 300 seconds
+# for register test, we run 300 with no nemesis
```
Comment (siddontang) [inline tidb/run.sh]:
```
we run:
1. 300 with no nemesis 
2. ...
3. ...
```

File: tidb/.gitignore [hunk 6]
```diff
@@ -9,3 +9,4 @@ pom.xml.asc
 /.nrepl-port
 .hgignore
 .hg/
+failed.log
```
Comment (siddontang) [inline tidb/.gitignore]: remove it

**Pull request merged**